views/js/qtiCreator/widgets/interactions/matchInteraction/states/Map.js

```diff
@@ -4,44 +4,47 @@ define([
     'taoQtiItem/qtiCreator/widgets/states/Map',
     'taoQtiItem/qtiCreator/widgets/interactions/matchInteraction/ResponseWidget',
     'lodash'
-], function($, stateFactory, Map, responseWidget, _){
+], function($, stateFactory, Map, responseWidget, _) {
 
-    var MatchInteractionStateMap = stateFactory.create(Map, function(){
+    return stateFactory.create(Map, function() {
 
-        var _widget = this.widget,
+        const _widget = this.widget,
             interaction = _widget.element,
-            response = interaction.getResponseDeclaration();
+            response = interaction.getResponseDeclaration(),
+            matchInteractionArea = $('.match-interaction-area input[type="checkbox"]', _widget.$container);
```

Comment:
Better to split by one declaration per variable:
```javascript
        const _widget = this.widget;
        const response = interaction.getResponseDeclaration();
        const matchInteractionArea = $('.match-interaction-area input[type="checkbox"]', _widget.$container);
```

And you can also take the chance to fix the code style issue with the name of the variable `_widget`. Names must not start with a `_`.

Reply:
Done, take a look if is better name :)